soc/xtensa/nxp_adsp/imx8m/Kconfig.defconfig.series

```diff
@@ -11,44 +11,20 @@ config SOC_TOOLCHAIN_NAME
 	string
 	default "nxp_imx8m_adsp"
 
-# if SOC_MIMX8M_ADSP
-
 config SOC
 	string
 	default "mimx8ml8"
 
-config SMP
-	default n
-
```

Comment:
Same here, I think this kind of changes should be placed in a separate patch with an explanation so that later when we go and try to understand the changes in git history everything will be simple and clear.

Reply:
Updated per your recommendation. Thank you!